pytest_datafiles.py

```diff
@@ -2,15 +2,20 @@
 Module containing a 'datafiles' fixture for pytest Tests.
 """
 
+import os
 from py import path  # pylint: disable=E0611
 import pytest
 
 
 def _copy(src, target):
     if not src.exists():
         raise ValueError("'%s' does not exist!" % src)
+
     if src.isdir():
         src.copy(target / src.basename)
+    elif src.islink():
+        linkto = os.readlink(str(src))
+        os.symlink(linkto, str(target / src.basename))
```

Comment:
These two lines could be replaced with `(target / src.basename).mksymlinkto(src.realpath())`

Reply:
Thanks, both good simplifications!